Formula/mqttx-cli.rb

```diff
@@ -0,0 +1,12 @@
+class MqttxCli < Formula
+  desc "Open source MQTT 5.0 CLI client and MQTT X on the command-line"
+  homepage "https://github.com/emqx/MQTTX/tree/main/cli"
+  url "https://github.com/emqx/MQTTX/releases/download/v1.8.0/mqttx-cli.tar.gz"
+  version "1.8.0"
+  sha256 "d9b8db76bb966fa5e9fcdc3b1fc3a17e3302c5dc9cb6f05e20eead814b37241a"
+  license "Apache"
+
+  def install
+    bin.install "mqttx"
+  end
+end
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hi, could you please provide the test case document? sorry about that I have no idea to do.

Reply:
Sorry I try it again.